packages/react/src/Tabs/Tabs.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { render, screen } from '@testing-library/react'
+import { createRef } from 'react'
+import { Tabs } from './Tabs'
+import '@testing-library/jest-dom'
+
+describe('Tabs', () => {
```

Comment:
Can and should we test keyboard behaviour?

Reply:
The keyboard behaviour is tested in packages/react/src/common/useFocusWithArrows.test.tsx